engine/src/game/game.ts

```diff
@@ -206,7 +209,26 @@ export class Game {
         }
 
         this._pixiSetup.onInterval(() => {
-            this._gameLoop.runStep(this.input, this._pixiSetup, this._physicsEngine);
+            const now = Date.now();
```

Comment:
I still think we should use `performance.now()`. While the precision is capped, it's more likely that `performance.now` is backed by a higher precision source than the system wall clock time. Furthermore, `performance.now` is supposed to be monotonic, so no weird jumps if user changes the system time or something crazy like that.

Reply:
`performance.now()` needs to be loaded from a different place in node, which would be a bit of a pain to set up. It's probably technically better, but:

- Handling weird jumps is something we need to do anyway — if the browser or something is minimized a big chunk of time is missing. Jumping backwards is weird, but if a user changes their system time to the distant past, they shouldn't be surprised that actively running processes freeze. If we're really concerned about that, we can handle it like we handle jumping forwards no problem.
- We don't need it to be too precise, just to the nearest 17 ms, which seems reasonable to expect of `Date.now()`.
- `Date.now()` worked fine when I tested it.